src/control-plane/backend/lambda/api/service/quicksight/reporting-utils.ts

```diff
@@ -818,4 +879,62 @@ export function getTempResourceName(resourceName: string, action: ExploreRequest
   }
 
   return resourceName;
-}
\ No newline at end of file
+}
+
+export function getDashboardTitleProps(analysisType: AnalysisType, query: any) : DashboardTitleProps {
```

Comment:
let's use lib `i18next` to fetch the i18n string from external translation files.

Reply:
means we need initialize a i18n context in backend code ?

Reply:
see https://github.com/i18next/i18next-http-middleware

Reply:
fixed